src/middleWares/auth.js

```diff
@@ -0,0 +1,20 @@
+import jwt from "jsonwebtoken";
+
+const verifyToken = (req, res, next) => {
+    const authHeader = req.headers.authorization;
+
+    if (!authHeader) {
+        return res.sendStatus(403);
+    }
+    const token = authHeader.split(" ")[1];
+
+    const user = jwt.decode(token, process.env.ACCESS_TOKEN_SECRET);
+    if (!user) {
+        return res.sendStatus(401);
+    }
+
+    req.user = user;
+    return next(), res.sendStatus(202);
```

Comment:
It would be better to send some good self-explanatory custom messages in the response

Reply:
Can you please be specific here ?